src/pybind/mgr/mgr_util.py

```diff
@@ -1,8 +1,18 @@
+import cephfs
 import contextlib
 import datetime
+import errno
 import os
 import socket
+import time
 import logging
+import sys
+from threading import Lock, Condition, Event
+from typing import no_type_check
+if sys.version_info >= (3, 3):
+    from threading import Timer
+else:
+    from threading import _Timer as Timer
```

Comment:
we're already on PY3 only.

Reply:
This was recently fixed in a611864dae86fcee9ea9cb9725aa26318396b55e. I saw no reason to rip that out.